src/hotspot/share/opto/node.cpp

```diff
@@ -664,55 +664,51 @@ void Node::destruct(PhaseValues* phase) {
   }
 }
 
-//------------------------------grow-------------------------------------------
-// Grow the input array, making space for more edges
-void Node::grow(uint len) {
+//------------------------------resize_array-------------------------------------------
```

Comment:
```suggestion
```
Just FYI. You can have these headers, but it is not expected any more. So I usually leave them out. But up to you.